src/Storage/Storage.Autorest/examples/Start-AzStorageAccountMigration.md

```diff
@@ -1,33 +1,22 @@
-### Example 1: Start a Storage account migration
+### Example 1: {{ Add title here }}
 ```powershell
-Start-AzStorageAccountMigration -AccountName myaccount -ResourceGroupName myresourcegroup -TargetSku Standard_LRS -Name migration1 -AsJob
+{{ Add code here }}
 ```
 
-This command starts a migration to Standard_LRS for Storage account myaccount under resource group myresourcegroup.
-
-### Example 2: Start a Storage account migration by pipeline
-```powershell
-Get-AzStorageAccount -ResourceGroupName myresourcegroup -Name myaccount | Start-AzStorageAccountMigration  -TargetSku Standard_LRS -AsJob
+```output
+{{ Add output here (remove the output block if the example doesn't have an output) }}
```

Comment:
Looks like the samples for Start-AzStorageAccountMigration got deleted... Can we add them back?